duxy/Day1/ej1.java

```diff
@@ -0,0 +1,36 @@
+import java.io.*;
+class ej1 {
+    public static void main(String[] args) {  
+        BufferedReader br = null;
+        try {
+            br = new BufferedReader(new FileReader("input.txt"));
```

Comment:
Use `Scanner` class instead of `BufferedReader` more easy and concise (more high level)